src/components/Menu.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Button, makeStyles, Container } from '@material-ui/core';
+import { Link } from 'react-router-dom'
+
+const useStyles = makeStyles((theme) => ({
+    topnav: {
+        backgroundColor: theme.palette.primary.main,
+        color: theme.palette.primary.contrastText,
+    },
+    topnavContent: {
+        display: 'flex',
+        height: 50,
```

Comment:
It's better to use percentage instead of pixels whenever possible, as this will scale up better with higher/lower resolution displays